Formula/pre-commit.rb

```diff
@@ -47,7 +47,16 @@ def post_install
             -   id: trailing-whitespace
       EOS
       system bin/"pre-commit", "install"
-      system bin/"pre-commit", "run", "--all-files"
+      (testpath/"f").write "hi\n"
+      system "git", "add", "f"
+
+      ENV["GIT_AUTHOR_NAME"] = "test user"
+      ENV["GIT_AUTHOR_EMAIL"] = "test@example.com"
+      ENV["GIT_COMMITTER_NAME"] = "test user"
+      ENV["GIT_COMMITTER_EMAIL"] = "test@example.com"
+      git_exe = `which git`.strip
```

Comment:
git_exe = which("git") should work I think

Reply:
indeed it does! updated!